### PR TITLE
Feature flag/fetch

### PR DESCRIPTION
**Pull request by akash2819**
Added api end point for feature flags

### CODE REVIEW

File: controllers/flag.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    logger.error(`Error while adding featureFlag info: ${err}`);
+    return res.boom.badImplementation("Something went wrong please contact admin");
+  }
+};
+const fetchFlags = async (req, res) => {
+  try {
+    const allFlags = await flagQuery.fetchFlags();
+    return res.json({
+      message: allFlags.length ? "Flags returned successfully!" : "No flag found",
+      flagQuery: allFlags,
```
Comment (pallabez) [inline controllers/flag.js:26]:
```suggestion
    const flags = await flagQuery.fetchFlags();
    return res.json({
      message: flags.length ? "Flags returned successfully!" : "No flag found",
      flags
```

Can we keep it accurate to the res.body? 
- https://github.com/Real-Dev-Squad/website-backend/issues/567

File: models/flag.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  }
+};
+
+const fetchFlags = async () => {
+  try {
+    const flag = await flagModel.get();
+    const flags = [];
+    flag.forEach((doc) => {
+      const flagdata = doc.data();
+      flag.push({
```
Comment (pallabez) [inline models/flag.js:25]: It should be `flags.push` right?

Comment (akash2819) [reply models/flag.js]: Yes ,  Sorry for that :(
Now resolved Please check

**Comment by prakashchoudhary07** [comment]
Closing this, as feature flag service is being maintained in other project
https://github.com/Real-Dev-Squad/feature-flag-service/ https://github.com/Real-Dev-Squad/feature-flag-backend/

**Pull request closed without merging**